File: app/src/main/java/com/xuie/android/ui/recycler/axis/AxisFragment.java
package com.xuie.android.ui.recycler.axis;


import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import androidx.annotation.NonNull;
import androidx.fragment.app.Fragment;
import androidx.recyclerview.widget.LinearLayoutManager;
import androidx.recyclerview.widget.RecyclerView;

import com.xuie.android.R;

/**
 * A simple {@link Fragment} subclass.
 * 这是一个时间轴的DEMO，添加的 {@link RecyclerView.ItemDecoration}样例
 *
 * @author xuie
 */
public class AxisFragment extends Fragment {

    @Override
    public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        View view = inflater.inflate(R.layout.fragment_axis, container, false);
        RecyclerView recyclerView = view.findViewById(R.id.recycler_view);
        recyclerView.setAdapter(new AxisAdapter());
        recyclerView.setLayoutManager(new LinearLayoutManager(getContext()));
        recyclerView.addItemDecoration(new AxisItemDecoration());
        return view;
    }

}

File: app/src/main/java/com/xuie/android/ui/recycler/axis/AxisFragment.kt
package com.xuie.android.ui.recycler.axis


import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import androidx.recyclerview.widget.LinearLayoutManager
import androidx.recyclerview.widget.RecyclerView

import com.xuie.android.R

/**
 * A simple [Fragment] subclass.
 * 这是一个时间轴的DEMO，添加的 [RecyclerView.ItemDecoration]样例
 *
 * @author xuie
 */
class AxisFragment : Fragment() {

  override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
    val view = inflater.inflate(R.layout.fragment_axis, container, false)
    val recyclerView = view.findViewById<RecyclerView>(R.id.recycler_view)
    recyclerView.adapter = AxisAdapter()
    recyclerView.layoutManager = LinearLayoutManager(context)
    recyclerView.addItemDecoration(AxisItemDecoration())
    return view
  }

}
